### PR TITLE
ENYO-3175: Added support for Mozilla based browsers

### PR DESCRIPTION
**Pull request by Djspaceg**
Enyo-DCO-1.1-Signed-off-by: Blake Stephens <blake.stephens@lge.com>

### CODE REVIEW

File: src/SpriteAnimation/SpriteAnimation.js [hunk 1]
```diff
@@ -493,8 +502,14 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	_generateKeyframe: function (percent, x, y) {
-		return (Math.ceil(percent*1000000) / 10000) +'%	{ -webkit-transform: translate3d('+ x +'px, '+ y +'px, 0);	transform: translate3d('+ x +'px, '+ y +'px, 0); }\n';
+	_generateKeyframe: function (percent, x, y, prefix) {
```
Comment (webOS101) [inline src/SpriteAnimation/SpriteAnimation.js]: Why not just change this method to generate all the prefixed methods at once and only call it once?  Or would that cause too big a change?

Comment (Djspaceg) [inline src/SpriteAnimation/SpriteAnimation.js]: That was actually how the first version of this PR looked, but it's unnecessary and wasteful to have -webkit- rules inside a @-moz-keyframes block, and vice versa. We also need standards-only blocks generatable for minimum rulesets. We Also need -webkit- prefixed block without -webkit- prefixed rules for the dozen versions of chrome that DO support standard transform but DON'T support @keyframe rule.

This creates the most efficient and minimally required source code.

File: src/SpriteAnimation/SpriteAnimation.js [hunk 2]
```diff
@@ -493,8 +502,14 @@ module.exports = kind(
...
+		var outStr = (Math.ceil(percent*1000000) / 10000) +'%	{ ';
+		if (prefix) {
+			// add the prefix and the standard
+			outStr+= prefix + 'transform: translate3d('+ x +'px, '+ y +'px, 0); ';
```
Comment (aarontam) [inline src/SpriteAnimation/SpriteAnimation.js]: What do you think about throwing `'transform: translate3d('+ x +'px, '+ y +'px, 0); ` into a var so we don't have to repeat the concatenation?

Comment (Djspaceg) [inline src/SpriteAnimation/SpriteAnimation.js]: Yeah, that seems fine.

**Pull request merged**